transip/service/dns.py

```diff
@@ -0,0 +1,50 @@
+"""
+DNS related objects
```

Comment:
There is a multitude of options ;-) But the best is probably to change the `.pylintrc` file. (See http://docs.pylint.org/ for more info)

The name of the rule that is violated is in braces on the end of the warning/error. e.g. `too-few-public-methods` for `R:  4, 0: Too few public methods (0/2) (too-few-public-methods)`. It's a bit annoying, but rules have names _and_ identifiers. Identifieres for rule-names can be found here: http://docs.pylint.org/features.html. (In this case it's `R0903`)

This rule has a threshold of minimum 2 public methods. It doesn't make sense to lower that to 1, so disabling this rule is probably the best way to go.

Change `.pylintrc` in the `[MESSAGES CONTROL]` section to add `R0903` to the `disable=` property.

Idem for the other error.